src/controls/dateTimePicker/DateTimePicker.tsx

```diff
@@ -205,7 +207,9 @@ export class DateTimePicker extends React.Component<IDateTimePickerProps, IDateT
       strings: dateStrings = new DateTimePickerStrings(), // Defines the DatePicker control labels
       timeDisplayControlType,
       placeholder,
-      showLabels
+      showLabels,
+      minDate = addYears(this.state.today, 1),
+      maxDate = addYears(this.state.today, 1)
```

Comment:
I don't think we need to set some values to `minDate` and `maxDate` by default.
Moreover, here both values are set to the same value.

Reply:
I've made the chage